docs/operation/enable_ip_ip.rst

```diff
@@ -0,0 +1,17 @@
+.. _enable IP-in-IP:
+
+Enable IP-in-IP encapsulation
+=============================
+
+.. _IP-in-IP: https://en.wikipedia.org/wiki/IP_in_IP
+
+By default Calico in MetalK8s is configured to use IP-in-IP_ encapsulation only
+for Cross Subnet communication.
+
+To Always use IP-in-IP_ encapsulation edit the IPPool used by Calico:
```

Comment:
```suggestion
To always use IP-in-IP_ encapsulation edit the ``IPPool`` used by Calico:
```

Reply:
Since we don't want to dive too deep in the internals of Calico for MetalK8s users, the "edit the IPPool used by Calico" could become "apply/run the following command"